rabbit-perf-test/Makefile

```diff
@@ -0,0 +1,2 @@
+build:
+	docker build -t eu.gcr.io/census-rm-performance/rm/census-rm-rabbit-performance . 
```

Comment:
I don't think census-rm-performance is the right project.
Also could be another command in the top level make file instead?

Reply:
Changed to CI.

Reply:
Moved build command to top level make file.